### PR TITLE
Use namespaces in XML import

### PR DESCRIPTION
**Pull request by maljac**
Resolves issue #250

### CODE REVIEW

File: opcua/common/xmlparser.py [hunk 1]
```diff
@@ -53,7 +53,8 @@ def __init__(self):
 
 class XMLParser(object):
 
-    def __init__(self, xmlpath):
+    def __init__(self, xmlpath, server):
+        self.server = server  # POC
```
Comment (maljac) [inline opcua/common/xmlparser.py]: @oroulet Not sure if you are happy with this.

Comment (oroulet) [inline opcua/common/xmlparser.py]: Send necessary to implement what you proposed. If it does not break things, should be ok

**Comment by maljac** [comment]
PyPy: 
test_datetime_read (tests_client.TestClient) ... FAIL

All other checks green...

**Comment by oroulet** [comment]
probably because the pypy server ran so slowly that it tok more than 1 second

**Comment by oroulet** [comment]
looks OK but one think I though of: The xml parser is used at startup to load standard address space, this is already very slow. Might this patch slow down startup time? Is so maybe you could disable check if namespace is 0? other ideas?

**Comment by maljac** [comment]
1. The checks only run if there are namespaces defined in the xml (Creating the namespaces and store them in a local dictionary)
2. I havent noticed any notable permformance problems while importing two XML models (~500 lines and ~6000 lines). 

I could add a additional check which would skip the regex check in `_get_node_id` if the namespace dict is empty. This would save the execution time for the pre-compiled regex.

**Comment by oroulet** [comment]
Ok

**Comment by DrMatthiasArnold** [comment]
I'm trying to import the OPC-UA Device companion standard node2.ml utilizing xml_import.
However, besides the DeviceSet object no additional type definitions etc are visible.
Any source code example which sheds some light on the usage of xml_import)

**Pull request merged**